classic/src/MessageBox.js

```diff
@@ -25,6 +25,18 @@ Ext.define('Zan.common.MessageBox', {
             if ('yes' === buttonId) deferred.resolve(true);
         });
 
+        // This title object will be created anyways in the Msgbox.confirm() method if title is a string.
+        // We are creating it here so we can customize fields like the buttons and defaultFocus
+        title = {
```

Comment:
I know the confirm method handles passing title as a config, but the documentation states it should be a string. 
Should we bypass the Ext.MessageBox.confirm method here and instead pass this config to Ext.MessageBox.show which is what confirm is doing?